src/aap_eda/tasks/ruleset.py

```diff
@@ -54,9 +54,10 @@ def activate_rulesets(
 
 @job
```

Comment:
should not this be @job("activation") too?

Reply:
@Alex-Izquierdo No, the `deactivate` has to be put into `default` queue, otherwise will be stuck by activation tasks.

Reply:
What do you think to add the @job("default") in these places? `explicit is better than implicit`
https://peps.python.org/pep-0020/